demo/demo.py

```diff
@@ -61,7 +73,7 @@ def __init__(self, ie,  model_path, device='CPU', ext_path=''):
     def get_detections(self, frame, detections):
         """Returns all detections on frame"""
         outputs = []
-        for rect in detections:
```

Comment:
Let's leave it as is. It's hard to understand why do we need pick the first element here without any contest.